### PR TITLE
Import client-python sources

### PR DESCRIPTION
**Pull request by vmax**

### CODE REVIEW

**Comment by lolski** [comment]
@haikalpribadi agreed. We should modularise it and I'd say we should spend some time looking into [CircleCI's Orbs functionality](https://circleci.com/docs/2.0/creating-orbs/):

_Orbs are reusable packages of CircleCI configuration that you may share across projects, enabling you to create encapsulated, parameterized commands, jobs, and executors that can be used across multiple projects_

**Comment by lolski** [comment]
@vmax please remove the `bazel-*` folders which got commited and add it to the `.gitignore` appropriately: `bazel-*`

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+              echo "Bazel will be executed with RBE support. This means the build is remotely executed and the cache will be re-used by subsequent CI jobs."
+              CMD="<< parameters.command >> --config=rbe"
+            else
+              echo "Bazel will be executed locally (without RBE support)."
+              CMD="<< parameters.command >>"
+            fi
+            echo "Executing $CMD"
+            $CMD
+
+  rpm_install:
```
Comment (lolski) [inline .circleci/config.yml]: @vmax do we need `rpm_install`?

Comment (vmax) [reply .circleci/config.yml]: @lolski we don't

**Comment by haikalpribadi** [comment]
> @haikalpribadi agreed. We should modularise it and I'd say we should spend some time looking into [CircleCI's Orbs functionality](https://circleci.com/docs/2.0/creating-orbs/)

Should we make a separate issue for this @vmax @lolski ?

**Comment by vmax** [comment]
@haikalpribadi we should

**Comment by haikalpribadi** [comment]
Can you make it and write the complete description - with problem and solution from above, @vmax ? Although, are we sure that Orbs is the absolute solution, @lolski ?

**Comment by vmax** [comment]
@haikalpribadi graknlabs/grakn#4840

**Pull request merged**